modules/apps/wiki/wiki-web/src/main/java/com/liferay/wiki/web/internal/helper/WikiAttachmentsHelper.java

```diff
@@ -31,18 +31,18 @@
 
 import javax.portlet.ActionRequest;
 
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Reference;
-
 /**
  * @author Roberto Díaz
  */
-@Component(service = WikiAttachmentsHelper.class)
 public class WikiAttachmentsHelper {
 
+	public WikiAttachmentsHelper(WikiPageService wikiPageService) {
```

Comment:
Hey @regiCesar21, transform the WikiAttachmentsHelper into a BaseMVCActionCommand and use it as a superclass for its usage. Thx